src/components/victory-bar.jsx

```diff
@@ -231,7 +231,21 @@ export default class VictoryBar extends React.Component {
       CustomPropTypes.allOfType([CustomPropTypes.integer, CustomPropTypes.nonNegative]),
       PropTypes.string,
       PropTypes.arrayOf(PropTypes.string)
-    ])
+    ]),
+    /**
+     * The Bar prop specifies a React component to use in building each bar in your chart.
+     * The value will be used in JSX.
+     * But, by default, Victory's standard Bar component will be used.
+     * @examples 'Bar', 'props => <Bar {...props} />'
+     */
+    Bar: PropTypes.func,
+    /**
+     * The BarLabel prop specifies a React component to use in building each label in your chart.
+     * The value will be used in JSX.
+     * But, by default, Victory's standard BarLabel component will be used.
+     * @examples 'BarLabel', 'props => <BarLabel {...props} />
+     */
+    BarLabel: PropTypes.func
```

Comment:
We already have a `labelComponents` prop, so we will need to reconcile that with `BarLabel`